resources/completion.bash

```diff
@@ -1,24 +1,29 @@
-# Copyright (c) 2021-present Fabien Potencier <fabien@symfony.com>
-#
-# This file is part of Symfony CLI project
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program. If not, see <http://www.gnu.org/licenses/>.
-#
-# Bash completions for the CLI binary
-#
-# References:
-#   - https://github.com/posener/complete/blob/master/install/bash.go
-#
+{{- /*
+Copyright (c) 2021-present Fabien Potencier <fabien@symfony.com>
+
+This file is part of Symfony CLI project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/ -}}
+
+{{- /*
+Bash completions for github.com/symfony-cli/console based projects
+
+References:
+  - https://github.com/posener/complete/blob/master/install/bash.go
+*/ -}}
+
+# Bash completions for {{ .App.HelpName }}
 
 complete -C "{{ .CurrentBinaryPath }} self:autocomplete" {{ .App.HelpName }}
```

Comment:
Shouldn't the filenames get their extension changed since they're not actually shell scripts, they're shell script templates?

Reply:
AFAIK, there's no official extension for Go template files. One of the most widespread uses of them is Helm chart templating, and it uses the targeted filename.

So I don't think it's worth the trouble to change them

Reply:
(Also, this could potentially break application that overrides them)